app/src/layouts/common/appSidebar/helpAndService/modal/requestSupportModal.jsx

```diff
@@ -19,20 +19,23 @@ import PropTypes from 'prop-types';
 import { reduxForm } from 'redux-form';
 import { useDispatch, useSelector } from 'react-redux';
 import { useIntl } from 'react-intl';
-import Parser from 'html-react-parser';
 import classNames from 'classnames/bind';
-import { withModal } from 'controllers/modal';
-import { ModalLayout } from 'components/main/modal';
-import { commonValidators } from 'common/utils/validation';
+import { bindMessageToValidator, commonValidators, validate } from 'common/utils/validation';
 import { userEmailSelector } from 'controllers/user';
 import { FieldProvider } from 'components/fields/fieldProvider';
 import { FieldErrorHint } from 'components/fields/fieldErrorHint';
-import { InputOutside } from 'components/inputs/inputOutside';
-import LoginIcon from 'common/img/login-field-icon-inline.svg';
 import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
 import { NOTIFICATION_TYPES, showNotification } from 'controllers/notification';
-import { HELP_AND_SUPPORT_EVENTS } from 'analyticsEvents/helpAndSupportEvents';
-import { messages } from '../messages';
+import { HELP_AND_SUPPORT_EVENTS } from 'components/main/analytics/events/ga4Events/helpAndSupportEvents';
+import { FieldText } from 'componentLibrary/fieldText';
+import { ModalLayout } from 'componentLibrary/modal';
+import { withModal } from 'components/main/modal';
```

Comment:
```suggestion
import { withModal } from 'controllers/modal';
```
It is better to depend on `controllers` directory here.
The `components/main/modal` will be removed in the future in favor of our new modal from ui-kit.